VisualStudio/src/CookingPatches.cs

```diff
@@ -9,39 +9,27 @@ internal class CookingPotItem_DoSpecialActionFromInspectMode
     {
         internal static bool Prefix(CookingPotItem __instance)
         {
-            //Implementation.Log("CookingPotItem -- DoSpecialActionFromInspectMode");
-            if (__instance.GetCookingState() == CookingPotItem.CookingState.Cooking) //patch does not apply while cooking
+            float waterAmount = __instance.m_LitersWaterBeingBoiled;
+            if (__instance.GetCookingState() == CookingPotItem.CookingState.Ready && waterAmount > 0) //patch does not apply while cooking
```

Comment:
This is, so the patch is only applied when the water is ready boiling and it is water. Will be later changed to work with only melted but not boiled water